lib/galaxy/datatypes/converters/uncompressed_to_gz.xml

```diff
@@ -0,0 +1,31 @@
+<tool id="CONVERTER_uncompressed_to_gz" name="Convert uncompressed file to compressed" hidden="true" version="1.0.0" profile="20.01">
+    <command><![CDATA[
```

Comment:
can you please add a dependency for gzip, bgzip etc ...

Reply:
Is it necessary? I tested it by using a local Galaxy instance.

Reply:
I think so, you have gzip installed locally that why it works.